pkg/routes/v1.go

```diff
@@ -147,6 +147,7 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
 			repostReactionGroup.GET("/:guild_id", h.GetRepostReactionConfigs)
 			repostReactionGroup.POST("", h.ConfigRepostReaction)
 			repostReactionGroup.DELETE("", h.RemoveRepostReactionConfig)
+			repostReactionGroup.POST("/start-stop", h.CreateConfigRepostReactionStartStop)
```

Comment:
`/start-stop`  seem not clean, can you separate the 2 APIs?

Reply:
i think no need, this is used for admin and they dont want to add config 2 times
<img width="512" alt="image" src="https://user-images.githubusercontent.com/39881166/195636738-537ea9e6-c0bd-4c88-a537-d71cf1c30583.png">